### PR TITLE
Sort content by popularity using Google Analytics

### PR DESCRIPTION
**Pull request by jhabdas**
Just came across the [`jekyll-ga`](https://github.com/developmentseed/jekyll-ga) generator plugin from the team that built [http://www.healthcare.gov](https://www.healthcare.gov/) and am adding it for posterity.

### CODE REVIEW

File: docs/_docs/plugins.md [hunk 1]
```diff
@@ -753,6 +753,7 @@ LESS.js files during generation.
 - [Jekyll::Paginate::Category](https://github.com/midnightSuyama/jekyll-paginate-category): Pagination Generator for Jekyll Category.
 - [AMP-Jekyll by Juuso Mikkonen](https://github.com/juusaw/amp-jekyll): Generate [Accelerated Mobile Pages](https://www.ampproject.org) of Jekyll posts.
 - [Jekyll Art Gallery plugin](https://github.com/alexivkin/Jekyll-Art-Gallery-Plugin): An advanced art/photo gallery generation plugin for creating galleries from a set of image folders. Supports image tagging, thumbnails, sorting, image rotation, post-processing (remove EXIF, add watermark), multiple collections and much more.
+- [jekyll-ga](https://github.com/developmentseed/jekyll-ga): A Jekyll plugin that downloads Google Analytics data and adds it to posts. Useful for making a site that lists "most popular" content. [Read the introduction](https://developmentseed.org/blog/google-analytics-jekyll-plugin/) post on the developmentSEED blog.
```
Comment (DirtyF) [inline docs/_docs/plugins.md]: AnaLytics -> Analytics

**Comment by pnn** [comment]
i don't think we should merge this until the gem is published as a rubygem

**Comment by pnn** [comment]
@jhabdas by the way, can you confirm this plugin works with the latest jekyll version(s)?

**Comment by crispgm** [comment]
@pnn as a matter of fact, lots of plugins listed are not published as a gem and even some of them are just gist codes.

**Comment by pnn** [comment]
@crispgm right, forgot about that. the plugin list will (hopefully) be replaced soon anyways

**Comment by DirtyF** [comment]
@pnn FYI https://github.com/jekyll/plugins has a type variable to distinguish gems from ruby scripts, but we index all types of plugins.

**Comment by DirtyF** [comment]
Thanks @jhabdas. 

@jekyllbot: merge +doc

**Comment by jhabdas** [comment]
@pnn sorry I wasn't able to confirm this works with the latest version of Jekyll. But, if not, the added exposure ought to help make that shift occur. Appreciate the merge folks!

**Pull request merged**